src/input.rs

```diff
@@ -0,0 +1,34 @@
+/// modeler Input event Type
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
+#[allow(non_camel_case_types)]
+#[allow(unused)]
+pub enum ModelerInputEventType {
+    /// For the first pen down event (pen touches the screen)
+    kDown,
```

Comment:
I don't know why the cpp `ink-stroke-modeler` lib named these `k*`, we don't have to follow that anymore. We can name them `Down`, `Move`, `Up`

Reply:
Done but needing some API changes on the note side that I'll do a little later